src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -2420,15 +2556,33 @@ private static TypeSymbolWithAnnotations InferResultNullability(BoundExpression
                     }
                     break;
 
-                case ConversionKind.Identity:
+                case ConversionKind.NoConversion:
                 case ConversionKind.DefaultOrNullLiteral:
```

Comment:
I didn't see a corresponding test change. Can you add?

Reply:
This is just an optimization, to avoid unnecessary calls to `GenerateConversion`. There shouldn't be any observable change.

---
In reply to: [190665424](https://github.com/dotnet/roslyn/pull/26356#discussion_r190665424) [](ancestors = 190665424)